tests/test_users.py

```diff
@@ -28,22 +31,21 @@ def override_get_db():
 
 app.dependency_overrides[get_db] = override_get_db
 
-client = TestClient(app)
+client = TestClient(app)  # Run the tests
```

Comment:
This line does not really run the tests. The idea is more to create a test client that is able to execute requests